Total Score: -1

####### encryption ########

Cases Failed: 0
Total Cases: 5

Warnings: 0

Compilation Errors: 0

Linker Errors: 0

Runtime Results:
Process ran successfully
################################

####### label_generator ########

Cases Failed: 0
Total Cases: 3

Warnings: 0

Compilation Errors: 0

Linker Errors: 0

Runtime Results:
Process ran successfully
################################

####### matrix_vector ########

Cases Failed: 0
Total Cases: 4

Warnings: 0

Compilation Errors: 0

Linker Errors: 0

Runtime Results:
Process ran successfully
################################

####### parse_int ########
--- reference
+++ student
@@ -11,4 +11,4 @@
 Case 5:
 Parsing failed
 Case 6:
-Parsing failed
+1024

Cases Failed: 1
Total Cases: 7

Warnings: 0

Compilation Errors: 0

Linker Errors: 0

Runtime Results:
Process ran successfully
################################

